tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -907,9 +907,9 @@ static void collectModuleDependencies(ModuleDecl *TopMod,
 
   auto ClangModuleLoader = TopMod->getASTContext().getClangModuleLoader();
 
-  ModuleDecl::ImportFilter ImportFilter;
-  ImportFilter |= ModuleDecl::ImportFilterKind::Public;
-  ImportFilter |= ModuleDecl::ImportFilterKind::Private;
+  ModuleDecl::ImportFilter ImportFilter = {
```

Comment:
And here

Reply:
That doesn't work because `ImportFilter` is modified below.